tests/integ/scala/test_file_operation_suite.py

```diff
@@ -286,6 +286,7 @@ def test_put_stream_with_one_file_twice(session, temp_stage, path1):
     second_result = session.file.put_stream(
         fd, f"{stage_with_prefix}/{file_name}", overwrite=False
     )
+    fd.close()
```

Comment:
precommit failed again after the earlier "fix" so I check code for all file `open()`s and found one without a `close()`

Reply:
Should we use `with` in the test, or maybe not using file stream but just `BytesIO`?

Reply:
We are using `with` is other tests except for two. First, to check if we support two uploads from the same `fd`, and second, to check if we raise error on closed `fd`.